staff/alex-barba/precourse/calculadora/style/style.css

```diff
@@ -0,0 +1,49 @@
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Oswald';
+  color: white;
+  text-align: center;
+  width: 100%;
+  background: linear-gradient(rgba(0, 0, 0, 0.8), rgba(0, 0, 0, 0.8)), url("calculator.jpg") center;
+  margin: 0 auto;
+
+}
+
+h1 {
+	color: #78D6B1;
+	font-size: 60px;
+}
+
+table {
+	margin: 0 auto;
+}
+
+input {
+	width: 378px;
+	font-size: 25px;
+	padding: 15px;
+	text-align: right;
+}
+
+button {
+	width: 100px;
+	font-size: 20px;
+	padding: 15px;
+	background: transparent;
+	color: white;
+}
+
+#doit {
+	height: 232px;
```

Comment:
Hey! Try with a height of 240px, maybe it looks more compact...

Great shortcut in main.js though!

Reply:
Thx!